lib/api/goal/common/Fingerprint.ts

```diff
@@ -41,8 +41,7 @@ export class Fingerprint
                 ...dependsOn: Goal[]) {
 
         super({
-            ...FingerprintDefinition,
-            ...getGoalDefinitionFrom(goalDetailsOrUniqueName, DefaultGoalNameGenerator.generateName("fingerprint")),
+            ...getGoalDefinitionFrom(goalDetailsOrUniqueName, DefaultGoalNameGenerator.generateName("fingerprint"), FingerprintDefinition),
```

Comment:
Is that definition still being used somewhere? Otherwise: delete.

Reply:
Why should I delete it? It is not exported and only there to make it easier to change and extend the descriptions etc.

Reply:
Ok, I see it